analysis/tc_coolstore_deps_quarkus.go

```diff
@@ -0,0 +1,1082 @@
+package analysis
+
+import (
+	"github.com/konveyor/go-konveyor-tests/data"
+	"github.com/konveyor/go-konveyor-tests/data/identity"
+	"github.com/konveyor/go-konveyor-tests/hack/addon"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var CoolstoreWithDepsQuarkus = TC{
+	Name:        "Coolstore Source and Deps analysis using Quarkus branch",
+	Application: data.CoolstoreQuarkus,
+	WithDeps: true,
+	Task:        Analyze,
+	Labels: addon.Labels{
+		Included: []string{
+			"konveyor.io/target=cloud-readiness",
+			"konveyor.io/target=jakarta-ee",
+		},
+	},
+	Identities: []api.Identity{
+		identity.TackleTestappPublicMaven,
+	},
+	Analysis: api.Analysis{
+		Effort: 7,
+		Issues: []api.Issue{
+			{
+				Category: "mandatory",
+				Description: "Local HTTP Calls",
+				Effort: 7,
+				RuleSet: "cloud-readiness",
+				Rule: "localhost-http-00001",
+				Incidents: []api.Incident{
+					{
+						File: "/shared/source/coolstore/src/main/resources/application.properties",
+						Line: 10,
+						Message: "The app is trying to access local resource by HTTP, please try to migrate the resource to cloud",
+					},
+				},
```

Comment:
With `cloud-readiness` and `jakarta-ee` we should see more results than this. There is something going on, but not sure if it's the test itself or something else. It looks as if the java provider wasn't starting up for some reason. This particular issue belongs to a rule that uses the built-in provider, not the java provider.